README.adoc

```diff
@@ -172,10 +173,17 @@ for SSL certificate validation
 Supported schemes are `http` and `https`.
 * `connection-timeout` sets the connection timeout in milliseconds
 * `read-timeout` sets the read timeout in milliseconds
+* `token` sets the main token to authenticate with vault.  This can also be set with the environment variable SPRING_CLOUD_VAULT_TOKEN
```

Comment:
Mentioning the environment variable fits good to the token authentication section.